src/index.js

```diff
@@ -38,6 +38,9 @@ const NotificationStore = {
             this.addNotification(notification)
         }
 
+    },
+    clear() {
+      this.state = []
```

Comment:
A good method to add. My only concern is whether the transitions work while setting the state directly to an empty array. If yes, then it's fine from my POV

Reply:
in the example the transitions work correctly